ddspipe_core/include/ddspipe_core/monitoring/producers/TopicsMonitorProducer.hpp

```diff
@@ -29,15 +29,9 @@
 #include <ddspipe_core/types/participant/ParticipantId.hpp>
 #include <ddspipe_core/types/topic/dds/DdsTopic.hpp>
 
-#if FASTRTPS_VERSION_MAJOR < 2 || (FASTRTPS_VERSION_MAJOR == 2 && FASTRTPS_VERSION_MINOR < 13)
-    #include <ddspipe_core/types/monitoring/topics/v1/MonitoringTopics.h>
-    #include <ddspipe_core/types/monitoring/topics/v1/MonitoringTopicsPubSubTypes.h>
-    #include <ddspipe_core/types/monitoring/topics/v1/MonitoringTopicsTypeObject.h>
-#else
-    #include <ddspipe_core/types/monitoring/topics/v2/MonitoringTopics.h>
-    #include <ddspipe_core/types/monitoring/topics/v2/MonitoringTopicsPubSubTypes.h>
-    #include <ddspipe_core/types/monitoring/topics/v2/MonitoringTopicsTypeObject.h>
-#endif // if FASTRTPS_VERSION_MAJOR < 2 || (FASTRTPS_VERSION_MAJOR == 2 && FASTRTPS_VERSION_MINOR < 13)
+#include <ddspipe_core/types/monitoring/topics/MonitoringTopics.hpp>
+#include <ddspipe_core/types/monitoring/topics/MonitoringTopicsPubSubTypes.hpp>
+#include <ddspipe_core/types/monitoring/topics/MonitoringTopicsTypeObjectSupport.hpp>
```

Comment:
I think that you only need to include the MonitoringTopics.hpp header. You don't use the rest of headers.

```suggestion

```